packages/react-scripts/scripts/init.js

```diff
@@ -12,6 +12,34 @@ var path = require('path');
 var spawn = require('cross-spawn');
 var pathExists = require('path-exists');
 var chalk = require('chalk');
+var execSync = require('child_process').execSync;
+
+function insideGitRepository() {
+  try {
+    execSync('git rev-parse --is-inside-work-tree', {stdio: 'ignore'});
+    return true;
+  } catch (e) {
+    return false;
+  }
+}
+
+function gitInit() {
+  try {
+    execSync('git --version', {stdio: 'ignore'});
+
+    if (insideGitRepository()) {
+      return false;
+    }
+
+    execSync('git init', {stdio: 'ignore'});
+    execSync('git add .', {stdio: 'ignore'});
+    execSync('git commit -m "chore: initial commit from create-react-app"', {stdio: 'ignore'});
```

Comment:
Can we just make this `Initial commit`? I don't think a lot of people are familiar with what "chore" means in this context, and many don't use this format.